### ISSUE TITLE
There are some errors in svg.js.d.ts

### ISSUE THREAD

**Issue by lykl** [svgdotjs/svg.js#1245]
When I was packing the production package, the editor gave me an error, so I could not pack normally.
Finally I found the error existed with the file.
```
class PointArray extends Array<ArrayXY> {
    constructor();
    constructor(array?: ArrayAlias<ArrayXY> | number[]);

    toLine(): LineAttr;
    at(pos: number): PointArray;
    transform(m: Matrix | MatrixLike): PointArray
    move(x: number, y: number): this;
    size(width: number, height: number): this;
    bbox(): Box;
    to(a: any): Morphable;
    toString(): string;
}
```
The PointArray class has an `at` method, while the base class Array&lt; ArrayXY&gt; The following also exists, which leads to errors.
The cause of the error should be a variable name conflict.
The same error occurs in the:
```
class PathArray extends Array<PathCommand> {
    constructor();
    constructor(d: ArrayAlias<PathCommand> | PathArrayAlias);

    move(x: number, y: number): this;
    size(width: number, height: number): this;
    equalCommands(other: PathArray): boolean
    morph(pa: PathArray): this
    at(pos: number): PathArray
    parse(array?: ArrayAlias<PathCommand> | PathArrayAlias): PathCommand[];
    bbox(): Box;
    to(a: any): Morphable
}
```

**Comment by Fuzzyma** [comment]
Can you try master?

**Comment by lykl** [comment]
> Can you try master?

It seems that the NPM version has not been updated.
The latest version is 3.1.1

**Comment by Fuzzyma** [comment]
Thats why I tell you: try master

**Comment by Stefan-Behrendt** [comment]
Could you please release a new version with the bug fix via NPM anyways? It's unsatisfying not being able to use the latest version in our `package.json` file and using a link to the GitHub master instead...

**Comment by larshp** [comment]
also see https://github.com/svgdotjs/svg.js/issues/1212, possible fix merged in https://github.com/svgdotjs/svg.js/pull/1223, but not released

**Comment by tobium** [comment]
pls release a new version with #1212

**Comment by tobium** [comment]
What can we do to publish a new release?

**Comment by smarza** [comment]
Please. 🙏

**Comment by larshp** [comment]
Personally, I started sponsoring [the maintainer](https://github.com/sponsors/Fuzzyma), in hope to see more activity, if you are using the library I suggest considering, so we can have sustainable open source.

Worst case I'll have to write a new library, or migrate to a different one, which is much more effort than a small sponsorship, or releasing this change, where I also implemented the fix(assuming it works), https://github.com/svgdotjs/svg.js/pull/1223

**Comment by Fuzzyma** [comment]
Sorry, I am quite busy. I released a new version (3.1.2) that hold all the latest changes
